src/shared/errors.js

```diff
@@ -6,11 +6,15 @@ const ERROR_MISSING_FILE = 'ERROR_FILE_DOES_NOT_EXIST';
 const ERROR_PYTHON_PROCESS = 'ERROR_PYTHON_PROCESS';
 const ERROR_PYTHON_UNSUPPORTED_VERSION = 'ERROR_PYTHON_UNSUPPORTED_VERSION';
 const ERROR_PYTHON_NOT_INSTALLED = 'ERROR_PYTHON_NOT_INSTALLED';
+const ERROR_EXECUTION_PROCESS = 'ERROR_EXECUTION_PROCESS';
+const UNKNOWN_PROGRAMMING_LANGUAGE = 'UNKNOWN_PROGRAMMING_LANGUAGE';
 
 module.exports = {
   ERROR_GENERAL,
   ERROR_MISSING_FILE,
   ERROR_PYTHON_PROCESS,
   ERROR_PYTHON_UNSUPPORTED_VERSION,
   ERROR_PYTHON_NOT_INSTALLED,
+  ERROR_EXECUTION_PROCESS,
+  UNKNOWN_PROGRAMMING_LANGUAGE,
```

Comment:
Maybe rename to `ERROR_UNKNOWN_PROGRAMMING_LANGUAGE` so that all errors start with "ERROR"